avocado/core/task/runtime.py

```diff
@@ -0,0 +1,35 @@
+class RuntimeTask:
+    """Task with extra status information on its life cycle status.
+
+    The :class:`avocado.core.nrunner.Task` class contains information
```

Comment:
Yes, good catch!

Reply:
Actually, no! :) The docstring never references itself, only the `avocado.core.nrunner.Task` class.

Reply:
It is a little bit confusing then.

"Tasks contains information that is necessary to describe its persistence and ***execution***. This class wraps... with extra information about its ***execution*** aspects."

Reply:
The text is confusing indeed... (but still true and accurate, because it really contains *extra* information about the execution aspects).  What about:

```
    Task with extra status information on its life cycle status.

    The :class:`avocado.core.nrunner.Task` class contains information
    that is necessary to describe its persistence and execution by itself.

    This class wraps a :class:`avocado.core.nrunner.Task`, with extra
    information about its execution by a spawner within a state machine.

```

Reply:
Yes, it is better.

Reply:
> Yes, it is better.

Force pushed with those docstring changes (and a commit message fix that was still referencing TaskInfo).